span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/SpanProcessingConfigServiceImpl.java

```diff
@@ -156,6 +222,47 @@ public void deleteExcludeSpanRule(
     }
   }
 
+  private List<ExcludeSpanRule> buildSystemLevelExcludeSpanRules(
+      List<? extends com.typesafe.config.ConfigObject> configObjectList) {
+    return configObjectList.stream()
+        .map(this::buildExcludeSpanRuleFromConfig)
+        .collect(Collectors.toUnmodifiableList());
+  }
+
+  @SneakyThrows
+  private ExcludeSpanRule buildExcludeSpanRuleFromConfig(
+      com.typesafe.config.ConfigObject configObject) {
+    String jsonString = configObject.render();
+    ExcludeSpanRule.Builder builder = ExcludeSpanRule.newBuilder();
+    JsonFormat.parser().merge(jsonString, builder);
+    return builder.build();
+  }
+
+  private Optional<ExcludeSpanRule> getSystemLevelExcludeSpanRule(String id) {
```

Comment:
```suggestion
  private Optional<ExcludeSpanRule> getSystemExcludeSpanRule(String id) {
```

Reply:
Can we drop the term `level` across and call them system exclude span rule? Similary, other type of rules become user exclude span rule